src/cbapi/psc/livequery/__init__.py

```diff
@@ -3,4 +3,5 @@
 from __future__ import absolute_import
 
 from .rest_api import CbLiveQueryAPI
-from cbapi.psc.livequery.models import Run, Result
+from cbapi.psc.livequery.models import Run, RunHistory, Result, DeviceSummary
```

Comment:
Looking at the other __init__.py files in PSC we use a singular import if there are too many for a line then we utilize the parenthesis to move it to another line. https://github.com/carbonblack/cbapi-python/blob/master/src/cbapi/psc/threathunter/__init__.py

Reply:
I agree, parentheses would be good here

Reply:
Sure, but how many is "too many"? There are only four imports on this line for 78 characters.

Reply:
They should be included in a singular import not split into multiple lines of the same import. I provided an example where we used the alternative format

Reply:
I saw that, and that's all well and good, and I can _put_ the parentheses in if you really want them, but you still haven't answered the question: How many imports on one line is "too many"?